app/ai/voice/agents/breeze_buddy/handlers/internal/hold_and_consult.py

```
@@ -0,0 +1,383 @@
+            f"[hold_and_consult] Outbound call initiated: "
+            f"{outbound_call_sid} from inbound {call_sid}"
+        )
+
+        # ── 11. Update outbound lead with actual call_id ───────────────
+        try:
+            await update_lead_call_id_by_id(outbound_lead_id, outbound_call_sid)
+            logger.info(
+                f"[hold_and_consult] Updated lead {outbound_lead_id} "
+                f"with call_id {outbound_call_sid}"
+            )
+        except Exception as update_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)